src/main/java/org/futureCollars/lesson5/task2/FizzBuzz.java

```diff
@@ -0,0 +1,21 @@
+package org.futureCollars.lesson5.task2;
+
+public class FizzBuzz {
+    public static void main(String[] args) {
+        for (int i = 1; i < 100; i++) {
+            if (isDivisible(i, 3)) {
```

Comment:
może wyciągnij to do jakiejś metody w stylu playFizzBuzz(int number) gdzie number to będzie konkretna wartość zmiennej i z pętli

Reply:
hmmm chyba nie tak, zobacz, sama nazwa playFizzBuzz(number) sugeruje że na zmiennej number zagrasz w gre fizzbuzz. i zagrasz w nią raz. a ty tutaj wrzuciłeś pętle. W pętli to my powinniśmy wywoływać tą metodę.